Package.swift

```diff
@@ -16,15 +16,17 @@ let package = Package(
         // Dependencies declare other packages that this package depends on.
         // .package(url: /* package url */, from: "1.0.0"),
         .package(url: "https://github.com/apple/swift-docc-plugin.git", from: "1.0.0"),
-        .package(url: "https://github.com/GigaBitcoin/secp256k1.swift", from: "0.12.2")
+        .package(url: "https://github.com/GigaBitcoin/secp256k1.swift", from: "0.12.2"),
+        .package(url: "https://github.com/jedisct1/swift-sodium.git", revision: "63240810df971557fe9badc557257bdfbfeb90a3")
```

Comment:
swift-sodium is a wrapper around the sodium C library, which is the one that the referenced Kotlin implementation uses as well.